daq_macros.py

```diff
@@ -1867,17 +1871,23 @@ def snakeRasterBluesky(rasterReqID, grain=""):
     if govStatus.exception():
       logger.error(f"Problem during start-of-raster governor move, aborting! exception: {govStatus.exception()}")
       return
-    raster_flyer.configure_detector(file_prefix=rasterFilePrefix, data_directory_name=data_directory_name)
-    raster_flyer.detector.stage()
+    #raster_flyer.configure_detector(file_prefix=rasterFilePrefix, data_directory_name=data_directory_name)
```

Comment:
I originally left them in so the code could eventually be made safe to merge into master (probably creating another if beamline==nyx check), but changing to the MD2 means that will never need to happen.

Reply:
But, since we do not plan to do that, I have now removed them. [24130d1](https://github.com/NSLS-II/lsdc/pull/314/commits/24130d1029ce6c5abd5f04bd3219700a5e9e08c6)